internal/cmd/list.go

```diff
@@ -138,5 +143,5 @@ func list(cli *CLI) error {
 		cli.Output("You have not been granted access to any active destinations")
 	}
 
-	return writeKubeconfig(destinations, grants)
+	return writeKubeconfig(identity, destinations, grants)
```

Comment:
Not directly related to your PR, but I am surprised we update the config as part of `infra list`.

Reply:
It's to make sure the local config reflects whatever a users most recent access is. It may change for #1820